net-misc/icaclient/icaclient-22.9.0.21-r1.ebuild

```diff
@@ -0,0 +1,240 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop wrapper xdg-utils
+
+DESCRIPTION="ICA Client for Citrix Presentation servers"
+HOMEPAGE="https://www.citrix.com/"
+SRC_URI="amd64? ( linuxx64-${PV}.tar.gz )
+	x86? ( linuxx86-${PV}.tar.gz )"
+
+LICENSE="icaclient"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="l10n_de l10n_es l10n_fr l10n_ja l10n_zh-CN"
+RESTRICT="mirror strip fetch"
+
+ICAROOT="/opt/Citrix/ICAClient"
+
+QA_PREBUILT="${ICAROOT#/}/*"
+
+RDEPEND="
+	app-crypt/libsecret
+	dev-libs/atk
+	dev-libs/glib:2
+	dev-libs/libxml2
+	media-fonts/font-adobe-100dpi
+	media-fonts/font-misc-misc
+	media-fonts/font-cursor-misc
+	media-fonts/font-xfree86-type1
+	media-fonts/font-misc-ethiopic
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/gst-plugins-base:1.0
+	media-libs/gstreamer:1.0
+	media-libs/libogg
+	media-libs/libjpeg-turbo
+	media-libs/libvorbis
+	media-libs/speex
+	net-libs/libsoup:2.4
+	net-libs/webkit-gtk:4
+	sys-apps/util-linux
+	sys-libs/libcxx
+	sys-libs/libcxxabi
+	sys-libs/zlib
+	virtual/krb5
+	virtual/libudev
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/gtk+:3
+	x11-libs/libX11
+	x11-libs/libXaw
+	x11-libs/libXcomposite
+	x11-libs/libXdamage
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXinerama
+	x11-libs/libXmu
+	x11-libs/libXrender
+	x11-libs/libXt
+	x11-libs/pango
+"
+DEPEND=""
+
+pkg_nofetch() {
+	elog "Download the client file ${A} from
+	https://www.citrix.com/downloads/workspace-app/"
```

Comment:
this is in here since a very long time, i poll that to find new releases and it usually works

just tried again and it worked. So maybe just you or a temporary problem. That is the upstream URL to find the packages.